CHANGELOG.md

```diff
@@ -5,7 +5,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+## [0.17.0] - 2016-08-23
 
+### Added
```

Comment:
This should probably be "Changed", rather than "Added", since it's just changing an existing method, rather than adding anything new.